all.json

```diff
@@ -9,6 +9,19 @@
 		"12345678987654321.link",
 		"16-b.it",
 		"162.222.226.37",
+		"v2.polkastarter.com",
```

Comment:
Ned to look at this one. Inconclusive on the url scan overview.

Reply:
You are right, I'm gonna delete it.

Reply:
in polkastarter itself a scam haha